askama_parser/src/node.rs

```diff
@@ -563,8 +563,7 @@ impl<'a> Macro<'a> {
 #[derive(Debug, PartialEq)]
 pub struct FilterBlock<'a> {
     pub ws1: Ws,
-    pub filter_name: &'a str,
-    pub args: Vec<Expr<'a>>,
+    pub filters: Expr<'a>,
```

Comment:
I propose you extract a `struct Filter<'a> { name: &'a str, args: Vec<Expr<'a>> }` from `Expr::Filter` (in a separate commit), and then give `filters` the type `Vec<Filter<'a>>` in this commit.

Reply:
:+1:

Reply:
(If we end up needing to insert in front, maybe `args` should be a `VecDeque`?)

Reply:
Considering we very likely always have less than 10 arguments and most of the time less than 2, I think the advantage is quite small. Could be interesting to perf-check.